lib/gax.js

```diff
@@ -58,6 +58,9 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {number} maxRetries - the maximum number of retrying attempts that
```

Comment:
Yeah. Also, use whatever NodeJS convention makes sense to discourage anyone from using it (like how we prefix Python identifiers with `_` to discourage use. We don't want `maxRetries` to be an official part of the GAPIC layer; we're just providing it as a feature for `google-cloud-node` to expose.